### DISCUSSION TITLE
Can't open nwb file

### DISCUSSION THREAD

**Discussion by rosullivan1226** [talmolab/sleap#1515] [Help!]
Hi my sleap project crashed and now it won't open--it is a nwb file

**Comment by roomrys** [comment]
Hi @rosullivan1226,

Do you have the original `.slp` file still? Are you able to open that file? Otherwise, I am curious where the `.nwb` file originated from (there might be some discrepancies in the reading/writing between softwares).

This might be a bit too hopeful, but do you have any messages in your terminal from the initial crash? No worries if not. Also, are there any messages in the terminal for when you try to open the `.nwb` file in SLEAP? If there are messages, please copy/paste the entire message starting at a few lines before the word `Traceback` appears.

Thanks,
Liezl
